src/InlineObjectCreator/widget/InlineObjectCreator.js

```diff
@@ -35,6 +44,7 @@ mendix.widget.declare('InlineObjectCreator.widget.InlineObjectCreator', {
 	postCreate : function(){
 		//houskeeping
 		logger.debug(this.id + ".postCreate");
+		dom.addCss('widgets/InlineObjectCreator/widget/ui/InlineObjectCreator.css');

```

Comment:
Dont need to explicitly add the fill CSS. (unless it a additional css file) DOwn side it that your widget CSS will change after the widget  initially loaded. So if the  style conflicts with the app style it will cause issue after you visited the page. (hard to find the cause of such an issue)

If the CSS is in the correct path it will automatically added to the widget runtime.
`InlineObjectCreator/widget/ui/InlineObjectCreator.css`
Please check paths are case sensitive.

Reply:
Done